modules/modules.nix

```diff
@@ -42,6 +42,7 @@ let
     (loadModule ./programs/bat.nix { })
     (loadModule ./programs/beets.nix { })
     (loadModule ./programs/browserpass.nix { })
+    (loadModule ./programs/calcurse.nix { })
```

Comment:
Since the calcurse package is labelled as only supporting Linux it is best to here include `condition = hostPlatform.isLinux;`.